### PR TITLE
Add image classification tutorial in AutoMM

### PR DESCRIPTION
**Pull request by bryanyzhu**
*Issue #, if available:*
NA

*Description of changes:*
A tutorial on how to use AutoMMPredictor to do image classification. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by szha** [comment]
Job PR-1869-1 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1869/1/index.html

**Comment by szha** [comment]
Job PR-1869-2 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1869/2/index.html

File: docs/tutorials/multimodal/beginner_image_cls.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+# AutoMM for Image Classification: Quick Start
```
Comment (zhiqiangdon) [inline docs/tutorials/multimodal/beginner_image_cls.md:1]: Need to unify the title format to use either `:` or `-`. Other tutorials seem to use `-`.

File: docs/tutorials/multimodal/beginner_image_cls.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+Now, we fit a classifier using AutoMM as follows:
+
+```{.python .input}
+from autogluon.multimodal import AutoMMPredictor
+predictor = AutoMMPredictor(label="label", path="./automm_imgcls")
+predictor.fit(
+    train_data=train_dataset,
+    time_limit=30, # seconds
+) # you can trust the default config, e.g., we use a `swin_tiny_patch4_window7_224` model
```
Comment (zhiqiangdon) [inline docs/tutorials/multimodal/beginner_image_cls.md:34]: We use `swin_base_patch4_window7_224` as the default.

Comment (bryanyzhu) [reply docs/tutorials/multimodal/beginner_image_cls.md]: Just changed

File: docs/tutorials/multimodal/beginner_image_cls.md (hunk 1 above)
Comment (bryanyzhu) [reply docs/tutorials/multimodal/beginner_image_cls.md]: Just changed

**Review by zhiqiangdon — APPROVED** [review]
LGTM

**Review by Innixma — CHANGES_REQUESTED** [review]
Blocking to ensure no merge prior to release

**Comment by bryanyzhu** [comment]
Cool, thank you. Let me rebase and trigger the CI.

**Review by Innixma — APPROVED** [review]
Unblocking, thanks!

**Comment by szha** [comment]
Job PR-1869-5 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1869/5/index.html
